### PR TITLE
Added insert operation

### PR DESCRIPTION
**Pull request by Joesepherus**
**Description**

### CODE REVIEW

File: src/pages/algorithms/b-trees/index.md [hunk 1]
```diff
@@ -17,8 +17,13 @@ Properties of B-Tree
...
 
-Search
+### Operations
+### Search
 Search is similar to search in Binary Search Tree. Let the key to be searched be k. We start from root and recursively traverse down. For every visited non-leaf node, if the node has key, we simply return the node. Otherwise we recur down to the appropriate child (The child which is just before the first greater key) of the node. If we reach a leaf node and don’t find k in the leaf node, we return NULL.
 
-Traverse
+### Insert 
+First we search and find out to which node the key should belong to and we insert it into it. Afterwards we look for and fix these problems:
+1. if the number of keys is too high, greter than t-1 then we move the middle key to the nodes parent. We do this recursively up until root. If the number of keys in the root is too high, then we make the middle key to be the new root of the whole tree and connect it to the the node it was in before.
```
Comment (scissorsneedfoodtoo) [inline src/pages/algorithms/b-trees/index.md]: Noticed a couple of spelling and grammatical errors here. Also, there were a couple of sentences that could be a bit more clear. Would you mind rewriting these while you fix the merge conflicts?

Comment (Joesepherus) [reply src/pages/algorithms/b-trees/index.md]: I'll look in to it, thanks.

Comment (Joesepherus) [reply src/pages/algorithms/b-trees/index.md]: damn couldn't find how to resolve it and accidentally closed it, can you help me out with it? Do I need to add another pull request or how do I resolve it...

Comment (scissorsneedfoodtoo) [reply src/pages/algorithms/b-trees/index.md]: @Joesepherus, sorry, just saw your comment. It seems like the branch for this PR has been deleted, so I'm not able to reopen it.

Probably the easiest thing to do now is to open another pull request, then you can make your additions to the most up-to-date version of the article. How did you originally submit this PR? From your local machine or through the Github web interface?

**Pull request closed without merging**